gum/backend-arm64/gumstalker-arm64.c

```diff
@@ -2060,6 +2109,7 @@ gum_exec_block_write_event_submit_code (GumExecBlock * block,
   beach_label = cw->code + 1;
 
   gum_arm64_writer_put_push_all_x_registers (cw);
+  gum_arm64_writer_put_push_all_q_registers (cw);
```

Comment:
We don't save registers here on x86, because that's what the opened prolog has already done. Is this a bug?